client/src/pages/Contact.jsx

```diff
@@ -164,7 +164,7 @@ function Contact() {
         )}
         <button
           type="submit"
-          aria-labelledby="submit-btn"
+          aria-label="Soumettre"
```

Comment:
as I said on Discord <button>Envoyer</button> will be read as "bouton enregistrer" so the aria-label is not necessary here

Reply:
Yes, I must have made a mistake, several changes I have made did not stick, sending fix again, sorry for that

Reply:
no worries, now you can just remove this aria-label and it will be all good